strategies/affected/src/main/java/org/arquillian/smart/testing/strategies/affected/ClassFileIndex.java

```diff
@@ -114,66 +112,59 @@ private void replaceVertex(JavaElement newClass) {
 
     private void updateJavaElementWithImportReferences(JavaElement javaElementParentClass,
         String[] imports) {
+
         for (String importz : imports) {
+
+            if (addImport(javaElementParentClass, importz)) {
+                if (filter.shouldBeIncluded(importz)) {
+                    JavaClass javaClass = builder.getClass(importz);
+                    if (javaClass != null) {
+                        updateJavaElementWithImportReferences(javaElementParentClass, javaClass.getImports());
+                    }
+                }
+            }
+        }
+    }
+
+    private boolean addImport(JavaElement javaElementParentClass, String importz) {
+
+        if (!importz.startsWith("java")) {
```

Comment:
No, I mean it could have but having in filter means that the import itself will be part of the graph, since it is not necessary to have java dependencies in graph it is done in this place.